spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -981,7 +981,7 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
             None
           }
 
-        case Like(left, right, _) =>
+        case Like(left, right, escapeChar) if escapeChar == '\\' =>
```

Comment:
I tihnk it would be better to have a check for escape char within the match and then call `withInfo` if the escape char is not `\\` so that we can give the user a specific error such as `escape char not supported` otherwise I think we fall through to the catchall logic and the user will get an error `like is not supported`, which could be confusing

Reply:
@andygrove I have made this change in https://github.com/apache/datafusion-comet/pull/478/commits/bfac08a2e8029464c1d7ec44bb46d9d33384a1ef. Please review.